pypesto/optimize/optimizer.py

```diff
@@ -790,3 +795,78 @@ def nlopt_objective(x, grad):
 
     def is_least_squares(self):
         return False
+
+
+class FidesOptimizer(Optimizer):
+    """
+    Global/Local optimization using the trust region optimizer fides.
+    """
+
+    def __init__(self,
+                 hessian_update: Optional[fides.HessianApproximation] =
+                 None,
+                 options: Optional[Dict] = None):
+        """
+        Parameters
+        ----------
+        options:
+            Optimizer options.
+
+        hessian_update:
```

Comment:
make it clearer that None means the objective's Hessian (/approximation) is used?

Reply:
👍